### PR TITLE
Fixing the version output and adding tests

### PR DESCRIPTION
**Pull request by jpricketMSFT**

### CODE REVIEW

File: src/tfvc/commands/commandhelper.ts [hunk 1]
```diff
@@ -7,6 +7,21 @@
 import { parseString } from "xml2js";
 
 export class CommandHelper {
+    public static SplitIntoLines(stdout: string, skipWarnings?: boolean): string[] {
+        let lines: string[] = stdout.replace(/\r\n/g, "\n").split("\n");
+        skipWarnings = skipWarnings === undefined ? true : skipWarnings;
+
+        // Ignore WARNings that may be above the desired lines
+        if (skipWarnings) {
+            let index = 0;
```
Comment (jeffyoung) [inline src/tfvc/commands/commandhelper.ts]: I'd love me a type on this

File: src/tfvc/commands/findworkspace.ts [hunk 2]
```diff
@@ -49,7 +50,7 @@ export class FindWorkspace implements ITfvcCommand<IWorkspace> {
         }
 
         // Find the workspace name and collectionUrl
-        const lines = stdout.replace("\r\n", "\n").split("\n");
+        const lines = CommandHelper.SplitIntoLines(stdout);
```
Comment (jeffyoung) [inline src/tfvc/commands/findworkspace.ts]: type here too?

File: src/tfvc/commands/getversion.ts [hunk 3]
```diff
@@ -23,8 +24,12 @@ export class GetVersion implements ITfvcCommand<string> {
     }
 
     public async ParseOutput(executionResult: IExecutionResult): Promise<string> {
-        const stdout = executionResult.stdout;
+        const lines = CommandHelper.SplitIntoLines(executionResult.stdout);
```
Comment (jeffyoung) [inline src/tfvc/commands/getversion.ts]: type-ola?  (missed it last time)

File: test/tfvc/commands/commandhelper.test.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        let lines: string[] = CommandHelper.SplitIntoLines(text);
+        assert.equal(lines.length, 5);
+        assert.equal(lines[0], "warning");
+        assert.equal(lines[1], "one");
+        assert.equal(lines[2], "two");
+        assert.equal(lines[3], "");
+        assert.equal(lines[4], "");
+    });
+
+    it("should verify SplitIntoLines - leave WARNings", function() {
```
Comment (jeffyoung) [inline test/tfvc/commands/commandhelper.test.ts]: Maybe add a test where there's a non-WARN line between two WARN lines?  Is that a possible scenario?

Comment (jpricketMSFT) [inline test/tfvc/commands/commandhelper.test.ts]: Not that I have seen WARNings all come before the actual output of the command. So we simply strip them from the top.

**Pull request merged**